wrapanapi/systems/openstack.py

```diff
@@ -906,6 +919,10 @@ def list_flavor(self):
         flavor_list = self.api.flavors.list()
         return [flavor.name for flavor in flavor_list]
 
+    def list_keypair(self):
+        keypair_list = self.api.keypairs.list()
+        return [keypair.name for keypair in keypair_list]
```

Comment:
I would combine both rows

Reply:
@izapolsk do you want me to change the rest? `list_volume` `list_network` and so on using same format. I am trying to  keep consistent.

Reply:
okay, leave it as is then. this isn't crucial.